qutebrowser/browser/webengine/darkmode.py

```diff
@@ -315,7 +338,9 @@ def _variant(versions: version.WebEngineVersions) -> Variant:
         except KeyError:
             log.init.warning(f"Ignoring invalid QUTE_DARKMODE_VARIANT={env_var}")
 
-    if versions.webengine >= utils.VersionNumber(6, 3):
+    if versions.webengine >= utils.VersionNumber(6, 4):
+        return Variant.qt_64
+    elif versions.webengine == utils.VersionNumber(6, 3):
```

Comment:
```suggestion
    elif versions.webengine >= utils.VersionNumber(6, 3):
```

otherwise this only matches 6.3.0

Reply:
Oh, good catch! Thanks!